src/Discord/Internal/Rest/Guild.hs

```diff
@@ -53,6 +53,8 @@ data GuildRequest a where
   --   'MANAGE_CHANNELS' permission. Returns a list of all of the guild's 'Channel'
   --   objects on success. Fires multiple Channel Update 'Event's.
   ModifyGuildChannelPositions      :: GuildId -> [(ChannelId,Int)] -> GuildRequest [Channel]
+  -- | Returns a list of active threads in a guild
+  ListActiveGuildThreads   :: GuildId -> GuildRequest ListActiveThreads
```

Comment:
Have you tested this works and parses what discord gives us?

Reply:
i have tested it, but please do so also

Reply:
yeah went back to look and noticed that i did not add an export for `ListActiveGuildThreads`
maybe `ListActiveThreads` should be moved to `Internal.Types.Guild` ?

Reply:
> i have tested it, but please do so also

adding an image of a parsed return
![obraz](https://github.com/discord-haskell/discord-haskell/assets/93288843/6eb4aa1e-d7f3-48bd-85e2-39750a5efd70)

Reply:
> yeah went back to look and noticed that i did not add an export for `ListActiveThreads` maybe it should be moved to `Internal.Types.Guild` ?

i think that would be best so im going to do that

Reply:
> yeah went back to look and noticed that i did not add an export for `ListActiveThreads` maybe it should be moved to `Internal.Types.Guild` ?

correction, not `Internal.Types.Guild` but `Internal.Types.Channel`

Reply:
I disagree with the movement; the List Active Threads structure is only ever used in the output of this one thing, so it should be defined and exported from this file as well; it's a request type.

Reply:
reverted changes and added an export in `Internal.Rest.Guild`